Sources/MediaCategoryCells/MediaCollectionViewCell.swift

```diff
@@ -18,6 +18,7 @@ class MediaCollectionViewCell: BaseCollectionViewCell {
     @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var descriptionLabel: UILabel!
     @IBOutlet weak var newLabel: UILabel!
+    @IBOutlet weak var thumbnailWidth: NSLayoutConstraint!
```

Comment:
Ideally, I believe the IBOutlet should be set as private.

Reply:
can do and will do for all then